developer-docs-site/docs/concepts/wallet-adapter-concept.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Wallet Adapter Concept"
+id: "wallet-adapter-concept"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Wallet Adapter Concept
+
+Aptos provides a [monorepo](https://github.com/aptos-labs/aptos-wallet-adapter) “Aptos Wallet Adapter” for wallet and dapps creators for easy development and smooth process in building projects on the Aptos network.
+
+The Aptos Wallet Adapter acts as a service between dapps and wallets and exposes APIs for dapps to interact with the wallets by following our [Wallet Standard](../guides/wallet-standard).
```

Comment:
@clay-aptos it relies on the change you had in https://github.com/aptos-labs/aptos-core/pull/5806

Reply:
Ack. Thanks!

Reply:
Merged.